src/com/aca/ArmineArzumanyan/homeWork_1/items/gadget/MemoryGadget.java

```diff
@@ -0,0 +1,16 @@
+package com.aca.armineArzumanyan.homeWork_1.items.gadget;
+
+
+public class MemoryGadget extends Gadget {
+    private final int rum;
```

Comment:
the name rum doesn't describe the meaning of this property correctly, please search RUM in google and see the results, are they related to your logic ?